### PR TITLE
Add pull-create integration test

### PR DESCRIPTION
**Pull request by typeless**

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

File: integrations/pull_create_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+	"github.com/stretchr/testify/assert"
+)
+
+func testPullCreate(t *testing.T, session *TestSession, user, repo, branch string) {
+	req := NewRequest(t, "GET", path.Join(user, repo))
+	resp := session.MakeRequest(t, req)
+	assert.EqualValues(t, http.StatusOK, resp.HeaderCode)
+
+	// Click the little green button to craete a pull
```
Comment (tsl0922) [inline integrations/pull_create_test.go:22]: s/craete/create

Comment (lunny) [reply integrations/pull_create_test.go:22]: @typeless really.

Comment (appleboy) [reply integrations/pull_create_test.go:22]: @tsl0922 Please make new PR to fix typo

**Pull request merged**